src/dotnet/APIView/APIViewWeb/Repositories/CosmosReviewRepository.cs

```diff
@@ -58,24 +58,35 @@ public async Task<ReviewModel> GetReviewAsync(string reviewId)
 
         public async Task<ReviewModel> GetMasterReviewForPackageAsync(string language, string packageName)
         {
-            var queryDefinition = new QueryDefinition("SELECT * FROM Reviews r WHERE " +
+            var reviews = await GetReviewsAsync(language, packageName, true);
+            ReviewModel review = null;
+            if (reviews.Count() > 0)
+            {
+                review = reviews.First();
+            }
+            return review;
+        }
+
+        public async Task<IEnumerable<ReviewModel>> GetReviewsAsync(string language, string packageName, bool isAutomatic)
```

Comment:
At this point, we might as well extend the other `GetReviewsAsync` with an additional argument.

Reply:
I would like to do this in separate PR if that's fine.